### ISSUE TITLE
my loss keep rising while using crf in join mode

### ISSUE THREAD

**Issue by Coder-XWJ** [keras-team/keras-contrib#506]
I have an excel data whose x_trian.shape == [3792,14] and label.shape == [3792] to be bi classified.I add five LSTM Sequential and a full-connect layer.At the top of the model is CRF.
First i reshape the data like x_train.reshape(-1, 1, 14) and y_train = y_train.reshape(3792,1,2).Then the CRF  is
        crf_layer = CRF(units=2,learn_mode='join')
	model.add(crf_layer)
	model.summary()
	model.compile(loss=crf_loss,optimizer='adam',metrics=[crf_viterbi_accuracy])
The loss keeps rising while trainning!The accuracy keeps 0.5 for 70 epochs,at this time loss is in decline. But when accuracy start to rise, loss start rising,too. why it should so?
